docs/report/context_variables.md

```diff
@@ -150,8 +150,16 @@ Each part object has access to a lot of context variables about the part. The fo
 
 | Variable | Description |
 |----------|-------------|
-| barcode | Brief payload data (e.g. for labels) |
-| item_count | Simply returns the number of stock items in this location. | 
+| barcode | Brief payload data (e.g. for labels). Example: {"stocklocation": 826} where 826 is the primary key|
+| description | The description of the location  |
+| icon | The name of the icon if set, e.g. fas fa-warehouse |
+| item_count | Simply returns the number of stock items in this location | 
+| name | The name of the location. This is only the name of this location, not the path | 
+| owner | The owner of the location if it has one. The owner can actually only be assigned in the admin interface | 
+| parent | The parent location one level up. Returns None if it is already the top most one | 
```

Comment:
Question: Isn't this a bit redundant? The parent location is already one level up